eng/tools/rush-runner.js

```diff
@@ -111,7 +111,7 @@ const getServicePackages = (baseDir, serviceDirs) => {
     const packageJsons = getPackageJsons(searchDir);
     for (const filePath of packageJsons) {
       const contents = JSON.parse(fs.readFileSync(filePath, "utf8"));
-      if (contents["sdk-type"] === "client" || contents["sdk-type"] === "mgmt" || contents["sdk-type"] === "perf-test") {
+      if (contents["sdk-type"] === "client" || contents["sdk-type"] === "mgmt" || contents["sdk-type"] === "perf-test" || contents["sdk-type"] === "utility") {
```

Comment:
Safe to remove I think now that we don't even have the problem of longer builds because of the reduced set of packages.

Thoughts?
@praveenkuttappan @sadasant @witemple-msft @ramya-rao-a @jeremymeng